.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Is this file needed? I think its a debugging thing and does not truly belong here